default_latex_headers/cmds.tex

```diff
@@ -22,28 +22,3 @@
 \newcommand{\doing}{\textcolor{blue}{{\large{$\blacklozenge$}}}}
 \newcommand{\checkme}{\textcolor{orange}{{\large{$\blacktriangleleft$}}}}
 \newcommand{\fixme}{\textcolor{red}{{\large{$\blacktriangleright$}}}}
-
-% some style preferences for code listings
-\definecolor{codegreen}{rgb}{0,0.6,0}
```

Comment:
let's leave the colors, these are kind of useful and a good example.  But leave removed the stuff below